custom_components/sscpoe/config_flow.py

```diff
@@ -145,19 +166,281 @@ def login():
+    
+            try:
+                discovered = await self.hass.async_add_executor_job(
+                    SSCPOE_local_search, bind_iface_param, ttl
+                )
+            except Exception as e:
+                LOGGER.exception("SSCPOE multicast discovery failed: %s", e)
+                errors["base"] = "multicast_discovery_failed"
+            else:
+                if not discovered:
+                    errors["base"] = "multicast_discovery_timeout"
+                else:
+                    self.local_devices = []
+                    web_candidates = []
+                    # Process each discovered device with normalization
+                    for dev in discovered:
+                         old_device = self._normalize_device_data({
+                             "add_method": "old",
+                             "sn": dev.get("sn"),
+                             "ip": dev.get("ip"),
+                             "V": dev.get("V"),
+                             "Active_state": dev.get("Active_state", "active"),
+                         })
+                         self.local_devices.append(old_device)
+    
+                         # Add to web candidate list if IP is set
+                         if dev.get("ip"):
+                             web_candidates.append({"ip": dev["ip"]})
+
+                     # Check web accessibility for all candidates
+                    if web_candidates:
+                         await self._check_web_accessibility(web_candidates)
+    
+                     # Add web devices to local_devices
+                    for candidate in web_candidates:
+                         if candidate.get("web_token"):
+                             ip = candidate["ip"]
+                             # Find existing device with this IP
+                             src_dev = next((d for d in self.local_devices if d["ip"] == ip and d["add_method"] == "old"), None)
+                             if src_dev:
+                                 web_device = self._normalize_device_data({
+                                     "add_method": "web",
+                                     "ip": ip,
+                                     "sn": src_dev["sn"],
+                                     "V": src_dev["V"],
+                                     "Active_state": "active",
+                                     "web_token": candidate["web_token"],
+                                     "user_password": SSCPOE_LOCAL_DEF_PASSWORD,
+                                 })
+                                 self.local_devices.append(web_device)
+
+                    return await self.async_step_local_select()
+
+            bind_default = bind_iface
+            ttl_default = ttl
+    
+        schema_dict: dict = {
+            vol.Required(CONF_BIND_INTERFACE, default=bind_default): vol.In(iface_choices),
```

Comment:
Use DEFAULT_BIND_INTERFACE instead of bind_default. Just remove bind_default at all.